model/weather.smithy

```diff
@@ -2,7 +2,10 @@ $version: "2"
 
 namespace example.weather
 
+use smithy.protocols#rpcv2Cbor
+
 /// Provides weather forecasts.
+@rpcv2Cbor
```

Comment:
I added this trait for development. I could keep it in or remove it to its own fixture model when I write the tests.

Reply:
I'd vote for creating a seperate fixture model for this though I do like seeing the generated diff here (which is why I also liked the more complete white_label model).